### PR TITLE
Updated CHANGELOG format

### PR DESCRIPTION
**Pull request by euanmillar**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -42,50 +55,177 @@
...
+- **Generate default address according to logged-in user's location** We have dropped support for the 'agentDefault' prop which was used as initial value for SELECT_WITH_DYNAMIC_OPTIONS fields. If you have not made any changes to address generation, then this should not affect you. If you have, you can refer to this PR to see how agentDefault has been deprecated in an example country: [https://github.com/opencrvs/opencrvs-farajaland/pull/978](https://github.com/opencrvs/opencrvs-farajaland/pull/978)
+- **Remove system admin UI items: Application, Certificates, User roles, Informant notifications** We have now moved to configuring these items away from the UI in favour of directly editing these from country configuration repository in code - specifically in application-config-default.ts.
```
Comment (Zangetsu101) [inline CHANGELOG.md:109]:
```suggestion
- **Remove system admin UI items: Application, User roles** We have now moved to configuring these items away from the UI in favour of directly editing these from country configuration repository in code - specifically in application-config-default.ts.
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -42,50 +55,177 @@
...
+
+Follow the descriptions in the migration notes to re-provision all servers safely.
+
+### New features
+
+- Introduced rate limiting to routes that could potentially be bruteforced or extracted PII from.
+- The login and client application loading experience has improved. A loading bar appears before the javaScript bundle has loaded and this transitions when fetching records.&#x20;
+- Development time logs are now much tidier and errors easier to point out. Production logging will still remain as is.&#x20;
+- Masked emails and phone numbers from notification logs.
+- Support for landscape certificate templates.
```
Comment (Zangetsu101) [inline CHANGELOG.md:140]: ~I thought this was for v1.6?~ Looks like it's already in v1.5. The milestone in this ticket needs to be updated: https://github.com/opencrvs/opencrvs-core/issues/6816

File: CHANGELOG.md [hunk 3]
```diff
@@ -42,50 +55,177 @@
...
+
+- Included an endpoint for serving individual certificates in development mode. This improves the developer experience when configuring certificates.
+- Removed logrocket refrences.
+- Enable gzip compression in client & login
+- Use docker compose v2 in github workflows
+- Added SMTP environment variables into the qa compose file to enable QA of SMTP servers.
+- In the certificate, the 'Place of Certification' now accurately reflects the correct location.
+- Groom's and Bride's name, printIssue translation variables updated [#124](https://github.com/opencrvs/opencrvs-countryconfig/pull/124)
+- Add query mapper for International Postal Code field
+- Add support for image compression configuration
```
Comment (Zangetsu101) [inline CHANGELOG.md:160]: This looks to be a duplicate of line 144

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,24 +1,37 @@
...
 
-- Certificate handlebar for registration fees `registrationFees` [#6817](https://github.com/opencrvs/opencrvs-core/issues/6817)
-- Logged in user details handlebar `loggedInUser` [#6529](https://github.com/opencrvs/opencrvs-core/issues/6529)
-- Supporting document fields can now be made required
-- If there is only one option in the document uploader select, then it stays hidden and only the upload button is showed with the only option being selected by default
+- **Major new feature** Description
+- Misc new feature
 
-- #### ElasticSearch reindexing
+### New content keys requiring translation
```
Comment (rikukissa) [inline CHANGELOG.md:18]: If in the upgrade instructions we guided the team to run `yarn extract:translations`, then I think they would get this same list as the output of the script and we wouldn't have to manually maintain it.

https://github.com/opencrvs/opencrvs-core/blob/develop/packages/client/src/extract-translations.ts#L192

There's also the `--write` option that automatically adds the items in the country config. How do we integrate this better to the upgrade process?

**Pull request merged**